### ISSUE TITLE
Resolver PDF files Zuid-Holland not working

### ISSUE THREAD

**Issue by joepio** [openstate/open-raadsinformatie#286]
Example file [here](https://id.openraadsinformatie.nl/448051.ttl):

isBasedOn: `https://staten.zuid-holland.nl/dsresource?objectid=51d446d7-632b-4307-9d6a-866f895d72bb`
sourceUrl: `https://api.openraadsinformatie.nl/v1/resolve/staten.zuid-holland.nl/dsresource?objectid=51d446d7-632b-4307-9d6a-866f895d72bb`

@aolieman thought it might have to do something with query parameters in the `isBasedOn`.

**Comment by aolieman** [comment]
> @aolieman thought it might have to do something with query parameters in the isBasedOn.

Indeed! I think that because of the error message from the resolver:

```
<Error>
    <Code>NoSuchKey</Code>
    <Message>The specified key does not exist.</Message>
    <Details>
        No such object: ori-static/staten.zuid-holland.nl/dsresource
    </Details>
</Error>
```

If the key shown is the actual key that's being used for the resolver lookup -- without the query parameter `objectid` -- I would start looking there for the cause of this issue.

**Comment by jurrian** [comment]
The problem seems to be url encoding, the resource exists and with encoded `=` and `?` it works:
https://storage.googleapis.com/ori-static/staten.zuid-holland.nl/dsresource%3Fobjectid%3D51d446d7-632b-4307-9d6a-866f895d72bb

I hope there is a way to have the redirect do the url encoding, which would save us some reindexing.

**Comment by jurrian** [comment]
Nginx redirection did not work so I fixed it in the code.

**Comment by joepio** [comment]
Fix is deployed, but not all data is reprocessed with fix.

**Comment by jorritb** [comment]
> Fix is deployed, but not all data is reprocessed with fix.

Some of them might be a victim of the Kafka MSG_SIZE_TOO_LARGE problem.

**Comment by jurrian** [comment]
> > Fix is deployed, but not all data is reprocessed with fix.
> 
> Some of them might be a victim of the Kafka MSG_SIZE_TOO_LARGE problem.

That is unrelated to this problem.

**Comment by jurrian** [comment]
Resolver URLs are now urlencoded and the nginx config has been changed to be able to handle urlencoded URLs. Zuid-Holland at this point seems the only municipality with this problem. Zuid-Holland has been reprocessed and the solution seems to work as old items are now being replaced by urlencoded resolver URLs. Somehow the mentioned URL in the first post has not been reprocessed yet. How that can happen is not clear, I have started the reprocess of Zuid-Holland again.
